WalletWasabi/Blockchain/Transactions/SmartTransaction.cs

```diff
@@ -109,7 +109,7 @@ public IReadOnlyCollection<IndexedTxIn> ForeignInputs
 
 	public IReadOnlyCollection<IndexedTxOut> ForeignOutputs
 	{
-		get
+		get 
```

Comment:
The format changes in `SmartTransaction.cs` are wrong and unnecessary, revert them please.

Reply:
Oh.. I didn't make any changes to that. I guess it is the pull I did from Wasabi master. 

I looked at the exact file on GitHub, and this change actually corresponds to what's there.

Should I still revert it?

Reply:
> Should I still revert it?

Yes.

Reply:
Yea.. Reverted

Reply:
There is still a one line change that should be reverted in this file.

Reply:
My bad. Removed it. 

Thank you @yahiheb